UnityProject/Assets/Scripts/Systems/Antagonists/Highlander.cs

```diff
@@ -10,6 +10,7 @@ public class Highlander : Antagonist
 		public override void AfterSpawn(ConnectedPlayer player)
 		{
 			player.Script.playerHealth.EnableFastRegen();
+			player.Script.characterSettings.Speech = Speech.Scotsman;
```

Comment:
just making sure, you want to force this being the only accent?

Reply:
Yes